deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -605,24 +605,21 @@ def predict_proba_on_generator(self, generator, transformers=[]):
     if not self.built:
       self.build()
     with self._get_tf("Graph").as_default():
-      with tf.Session() as sess:
-        saver = tf.train.Saver()
-        saver.restore(sess, self.last_checkpoint)
-        out_tensors = [x.out_tensor for x in self.outputs]
-        results = []
-        for feed_dict in generator:
-          feed_dict = {
-              self.layers[k.name].out_tensor: v
-              for k, v in six.iteritems(feed_dict)
-          }
-          feed_dict[self._training_placeholder] = 1.0  ##
-          result = np.array(sess.run(out_tensors, feed_dict=feed_dict))
-          if len(result.shape) == 3:
-            result = np.transpose(result, axes=[1, 0, 2])
-          if len(transformers) > 0:
-            result = undo_transforms(result, transformers)
-          results.append(result)
-        return np.concatenate(results, axis=0)
+      out_tensors = [x.out_tensor for x in self.outputs]
+      results = []
+      for feed_dict in generator:
+        feed_dict = {
+            self.layers[k.name].out_tensor: v
+            for k, v in six.iteritems(feed_dict)
+        }
+        feed_dict[self._training_placeholder] = 1.0  ##
+        result = np.array(self.session.run(out_tensors, feed_dict=feed_dict))
+        if len(result.shape) == 3:
```

Comment:
Should we take the chance to remove the transpose here?

Reply:
That's unrelated to this PR.  The only thing that changed about those lines was the indentation.